<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostGameDialog</class>
 <widget class="QDialog" name="HostGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="titleEdit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thygrrr's Game</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="passEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>password</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="cursorPosition">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lobby Preview</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>6</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="passCheck">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioPublic">
       <property name="text">
        <string>Public</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioFriends">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Friends</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioHidden">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Hidden</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="optionGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Game Title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QListWidget" name="gamePreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>120</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Map (can be changed later)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="mapList"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="hostButton">
     <property name="text">
      <string>Host Game</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QListWidget" name="modList">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>titleEdit</zorder>
  <zorder>passEdit</zorder>
  <zorder>passCheck</zorder>
  <zorder>mapList</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>optionGroup</zorder>
  <zorder>gamePreview</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>hostButton</zorder>
  <zorder>modList</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>passCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>passEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
